src/libcharon/plugins/kernel_super/kernel_super_ipsec.c

```diff
@@ -185,6 +191,28 @@ CALLBACK(terminator_ka_timer_cb, job_requeue_t,
 	return JOB_RESCHEDULE_MS(TERMINATORS_KA_TIMER_MS);
 }
 
+CALLBACK(terminator_count_tunnels_timer_cb, job_requeue_t, 
+		struct terminator_count_tunnels_timer_cb_ctx* ctx) {
+	struct pm_terminator_conn* term_conn = NULL;
+	
+	private_kernel_super_ipsec_t* this = ctx->this;
+	const uint64_t term_unique_id = ctx->term_unique_id;
+	term_conn = terminators_table_find_by_unique_id(this->ipsec_terms, term_unique_id);
+	LOG(DBG_KNL, SL_LOG_INFO, "term_conn: %p", term_conn);
+
+	if (! term_conn) {
+		LOG(DBG_KNL, SL_LOG_ERR, "no terminator with unique ID %" PRIu64 " is present. Maybe terminator was deleted?",
+			term_unique_id);
+		return JOB_REQUEUE_NONE;
+	}
+
+	if(! count_tunnels(this, term_conn)) {
+		LOG(DBG_KNL, SL_LOG_ERR, "Failed to count tunnels on terminator: %u ", picomsg_terminator_conn_get_id(term_conn));
```

Comment:
No need to bombard the logs with this extra error log. If `count_tunnels()` failed it will log itself, no need for this log on top of it